lms/djangoapps/branding/api.py

```diff
@@ -218,12 +221,29 @@ def _build_support_form_url(full_path=False):
     return contact_us_page
 
 
+def _build_help_center_url(language=settings.LANGUAGE_CODE):
+    """
+    Return the help-center URL based on the
+    language selected on the homepage.
+    :param language: selected language
+    :return: help-center URL
+    """
+    support_url = settings.SUPPORT_SITE_LINK
+    enabled_languages = {
+        'en': 'hc/en-us',
+        'es': 'hc/es-419'
+    }
+    if language in enabled_languages:
+        support_url += '/' + enabled_languages[language]
+    return support_url
+
+
 def _footer_connect_links(language=settings.LANGUAGE_CODE):
     """Return the connect links to display in the footer. """
     links = [
         ("blog", (marketing_link("BLOG"), _("Blog"))),
         ("contact", (_build_support_form_url(full_path=True), _("Contact Us"))),
-        ("help-center", (settings.SUPPORT_SITE_LINK, _("Help Center"))),
+        ("help-center", (_build_help_center_url(settings.LANGUAGE_CODE), _("Help Center"))),
```

Comment:
I believe you need to pass `language` here instead of `settings.LANGUAGE_CODE`.

Reply:
Changed `settings.LANGUAGE_CODE` with `language`.